### ISSUE TITLE
Bug: dropdown traps focus

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/calcite-components#875]
### Summary

A simple dropdown captures the focus and prevents using the tab to move among components.

### Actual Behavior

When the focus is on the dropdown, tabs simply cycle among the hidden dropdown items and the dropdown button--a tab never moves on to the content after the dropdown. Shift-tab is a little bit different: tabs move through the dropdown's hidden items until the dropdown button is reached, and then control moves to the focusable content before the dropdown.

### Expected Behavior

One would expect to tab from button to dropdown to button, only entering the dropdown if Enter is used while the focus is on the dropdown. Shift-tab should be the same, except going backwards through the focusable content.

### Reproduction Steps

Illustrative web page:

![image](https://user-images.githubusercontent.com/2125181/90442546-b16edf80-e08f-11ea-9f69-4e5204a92d90.png)
built using the [dropdown-simple example](https://esri.github.io/calcite-components/?path=/story/dropdown--simple):

``` html
<!doctype html>
<html lang="en" dir="ltr">
<head>
  <link rel="stylesheet" href="./css/calcite.css" type="text/css" />
  <script type="module" src="src/lib/calcite-components/calcite.esm.js" crossorigin="use-credentials"></script>
</head>
<body>
  <button>Button before</button>

  <!-- https://esri.github.io/calcite-components/?path=/story/dropdown--simple -->
  <calcite-dropdown>
    <calcite-button slot="dropdown-trigger">Open Dropdown</calcite-button>
    <calcite-dropdown-group>
      <calcite-dropdown-item>Relevance</calcite-dropdown-item>
      <calcite-dropdown-item active>Date modified</calcite-dropdown-item>
      <calcite-dropdown-item>Title</calcite-dropdown-item>
    </calcite-dropdown-group>
  </calcite-dropdown>

  <button>Button after</button>
</body>
</html>
```

### Relevant Info 

Introduced with commit e434c48cee83658ee1e60a26a1f8f6f8c26349f8; its predecessor 0303150e195450e759a92a727137ba4754788f76 is OK.

Desktop Chrome 84, Firefox 79

**Comment by macandcheese** [comment]
I think this is a regression that popped up at some point recently - related: https://github.com/Esri/calcite-components/issues/778

**Comment by macandcheese** [comment]
Removing the 
```
[hydrated] {
  visibility: visible;
}
```
introduced here:
https://github.com/Esri/calcite-components/commit/e434c48cee83658ee1e60a26a1f8f6f8c26349f8#diff-d179153398bd7e202417508f8b373a8fR25-R27
solves it. I'm not sure if that was added for another purpose or if it may cause other tab interactions like this one... @driskull I ran our tests without this here and everything still passes. This was done in a version bump PR so was this to address some Stencil change?

**Comment by MikeTschudi** [comment]
Reproducible with the above example and @esri/calcite-components@1.0.0-beta.38

**Comment by macandcheese** [comment]
Seems to be working: https://jsbin.com/gagucukike/edit?html,js,output

**Comment by MikeTschudi** [comment]
Thanks, Adam.  We must have build problems.  Works for me when I use unpkg to get .38 or .39; doesn't work with our node_modules .38.
